build.sh

```diff
@@ -15,7 +18,7 @@ else
 fi
 
 rpm-ostree install \
-    akmod-${NVIDIA_PACKAGE_NAME}*:${NVIDIA_MAJOR_VERSION}.*.fc${RELEASE} \
+    akmod-${NVIDIA_PACKAGE_NAME}-${NVIDIA_MAJOR_VERSION}.*.fc${RELEASE} \
```

Comment:
Will need to confirm these with the older packages.  Works for v530 and v535, but not checked with the others